src/services/iasql.ts

```diff
@@ -908,3 +908,25 @@ export async function uninstall(moduleList: string[], dbId: string, orm?: Typeor
   }
   return "Done!";
 }
+
+export async function getStackInfo(dbId: string, stackName: string, ormOpt?: TypeormWrapper) {
+  let orm: TypeormWrapper | null = null;
+  try {
+    orm = !ormOpt ? await TypeormWrapper.createConn(dbId) : ormOpt;
+    // Find all of the installed modules, and create the context object only for these
+    const moduleNames = (await orm.find(IasqlModule)).map((m: IasqlModule) => m.name);
```

Comment:
why does the CF stack need to know what are the installed modules? to generate the permissions/policy? if so, installing new modules could get troublesome?

Reply:
The comments are not updated, just copy&paste from `apply`, I think. Just copying the pattern to get the `aws_account` module, get its context and being able to call the AWS API  with the new credentials. This is only to call CloudFormation to get the newly created stack and show the details to the users.